src/v/cloud_storage_clients/xml_sax_parser.cc

```diff
@@ -277,9 +286,7 @@ void abs_parse_impl::handle_characters(std::string_view characters) {
               characters.data(), characters.size());
         }
         break;
-    case xml_tag::next_continuation_token:
-        _items.next_continuation_token = {characters.data(), characters.size()};
-        break;
+    case xml_tag::is_truncated:
```

Comment:
could use attribute ``` [[fallthrough]]; ``` in the body of this case to make it explicit

Reply:
thanks, fixed